digicampipe/io/hessio_digicam.py

```diff
@@ -0,0 +1,297 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Components to read HESSIO data.
+
+This requires the hessio python library to be installed
```

Comment:
Is this part of the requirements now? or does it install with ctapipe?

Reply:
Also seems like the entire file is from ctapipe. What can be imported and not duplicated?

Reply:
> Is this part of the requirements now? or does it install with ctapipe?
Everything what it needs is installed with ctapipe.

> Also seems like the entire file is from ctapipe. What can be imported and not duplicated?
Correct, but the problem is that digicampipe needs some more containers to be filled than original ctapipe.io.hessio supports and since it is written like several long functions, I just took the functions and added a few more lines inside each of them.